spec/controllers/rents_controller.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+def mock_sign_in(user)
+  allow_any_instance_of(described_class).to receive(:authenticate_user!)
+  allow_any_instance_of(described_class).to receive(:current_user).and_return(user)
+end
+
+describe BooksController, type: :controller do
+  context 'when user is logged in' do
```

Comment:
Like we talked, also test for when the user isn't logged in (this should make the transition to pundit easier, too)